sycl/test-e2e/InvokeSimd/Spec/simd_size/simd32_platform_error.cpp

```diff
@@ -1,30 +0,0 @@
-// Test not intended to run on PVC
-// UNSUPPORTED: gpu-intel-pvc
```

Comment:
can you explain why this test was removed? do we not want to error for simd32 on unsupported platforms anymore?

Reply:
discussed offline